index.md

```diff
@@ -6,30 +6,33 @@ title: Documentation
 Introduction
 ============
 
-There are a number of sections in the documentation,
- offering help for the [kit](/docs/kit/) and [programming](/docs/programming/).
-Under the [tutorials](/docs/tutorials/) section,
- a number of these things are combined to help you understand what you can,
- or need, to do.
- Navigation of the documentation can be done using the column to the left,
- where everything is arranged alphabetically in the aforementioned sub-sections.
+This documentation explains how to use the kit and the robot's Python API.
+The information is spread across multiple sections:
 
+- The [kit](/docs/kit/) section will give you an overview of the physical kit that is provided to your team, how to connect to each board and what features they have.
+- The [programming](/docs/programming/) section talks through how to write code that interacts with all the boards.
+- The [rules](/docs/rules/) section is important as it talks through the aim of this year's game and what task you are trying to achieve.
+- [Tutorials](/docs/tutorials/) are a series of guides that will help you get started.
 
 Within this documentation, you will come across a number of boxes like this:
 
 ~~~~~ python
 # code example
 ~~~~~
 
-These are code examples provided to help you.
+These are pieces of example code that you can run on your robot.
 
-From time to time,
- you may come across some warnings such as the following:
+From time to time, you may come across some warnings such as the following:
 
-<div class="warning">Charge Your Batteries!</div>
+<div class="warning">Never leave batteries unattended when they are in use or charging.</div>
 
-It would be advisable to take note of these,
- especially that one! You will also come across some blue boxes providing information,
- similar to the following:
+It would be advisable to take note of these, especially that one!
```

Comment:
Can we note that these are often for safety.

Reply:
9ddbebe